trojsten/locale/sk/LC_MESSAGES/django.po

```diff
@@ -833,6 +833,21 @@ msgstr ""
 "Nahral si svoj popis riešenia po termíne kola. Nebude zarátaný vo "
 "výsledkovke."
 
+msgid "You have successfully submitted your solution."
+msgstr "Úspešne sa ti podarilo submitnúť riešenie."
+
+msgid ""
+"You have submitted your solution after the deadline. It is not counted in "
+"results."
+msgstr ""
+"Nahral si svoje riešenie po termíne kola. Nebude zarátané vo výsledkovke."
+
+msgid "Your solution is correct. You gain %(points)d points."
+msgstr "Tvoje riešenie je správne. Získavaš %(points)d bodov."
+
+msgid "Your solution is incorrect. You gain %(points)d points."
+msgstr "Tvoje riešenie je nesprávne. Získavaš %(points)d bodov."
```

Comment:
Ešte čo mi tak udrelo do očí je, že to neskloňuje počet bodov. Teda, vypíše to, že získavam 1 bodov, 2 bodov, atď. Možno by som to zmenil na "Tvoj počet bodov: {}", ale aj to mi príde trochu divné.

Reply:
nemáme niekde funkciu čo to skloňuje?

Reply:
spravil som rýchli search a zjavne to nemáme nikde, teda iba v nejakej praskovej úlohe to bolo ošetrené https://github.com/trojsten/web/blob/6bc22e7a20fcf963733ae0d3b00abb7dfd435caf/trojsten/special/plugin_prask_2_4_1/tester.py#L88

Reply:
To sa nerobí takto, v tom .po súbore existuje spôsob, ako tam dať viac prekladov pre rôzne formy plurálu. Viď napríklad https://github.com/trojsten/web/blob/1376ddb66c7eb6c0ce72c03e770a818a2602a4eb/trojsten/locale/sk/LC_MESSAGES/django.po#L946
#, python-format
msgid "%(count)d day"
msgid_plural "%(count)d days"
msgstr[0] "%(count)d deň"
msgstr[1] "%(count)d dni"
msgstr[2] "%(count)d dňa"
msgstr[3] "%(count)d dní"
U mňa to napríklad nefungovalo, ale na Bete a reálnom serveri už áno.

Reply:
neviem čo s tým teda, ako to opraviť

Reply:
Do toho prekladového .po súboru daj:

msgid "Your solution is incorrect. You gain %(points)d point."
msgid_plural "Your solution is incorrect. You gain %(points)d points."
msgstr[0] "Tvoje riešenie je nesprávne. Získavaš %(points)d bod."
msgstr[1] "Tvoje riešenie je nesprávne. Získavaš %(points)d body."
msgstr[2] "Tvoje riešenie je nesprávne. Získavaš %(points)d bodu."
msgstr[3] "Tvoje riešenie je nesprávne. Získavaš %(points)d bodov."

Analogicky pre správe riešenie.

Reply:
Keď to chceš potom použiť, tak takto:

from django.utils.translation import ngettext

ngettext("Your solution is incorrect. You gain %(points)d point.", "Your solution is incorrect. You gain %(points)d points.", points) % {"points": points}